libraries/AP_Scripting/applets/plane_precland.lua

```diff
@@ -0,0 +1,195 @@
+--[[
+ support precision landing on quadplanes
+
+ This is a very simple implementation intended to act as a framework
+ for development of a custom solution
+--]]
+-- luacheck: only 0
```

Comment:
This is cheating  ;)

Reply:
fixed, was copied from ship landing, need to fix that